src/agentscope/agents/agent.py

```diff
@@ -100,9 +100,9 @@ def __init__(
         host: str = "localhost",
         port: int = None,
         max_pool_size: int = 8192,
-        max_timeout_seconds: int = 1800,
+        max_timeout_seconds: int = 7200,
         local_mode: bool = True,
-        lazy_launch: bool = True,
+        lazy_launch: bool = False,
```

Comment:
Why we disable the lazy_launch here?

Reply:
This feature is not used at all and may be removed in future versions.